### DISCUSSION TITLE
no test suites found when i use vitest cli

### DISCUSSION THREAD

**Discussion by 1714080902120** [vitest-dev/vitest#2215] [Q&A]
I had installed vitest and vite in global, but when i use vitest in terminal like `vitest run -t test-id`, it did not work.
Here is the error:
![image](https://user-images.githubusercontent.com/48143943/198277584-5e105b44-d378-4ee7-abdc-46658f0327d7.png)

It can found the package and the file which contain the test suite， but just can't found the suite.
The command can successfully run when i config in package.json.
Is this a bug? Or i need to config something else ?
I had tried npm and pnpm, but still not work.
my node version is 16.15.1, my vitest version is 0.24.3 and my vite version is 3.2.0.

**Comment by sheremet-va** [comment]
Your test file `D:/vscode/vitest-study/tests/index.spec.js` doesn't have any `test`/`it` blocks

**Reply by 1714080902120** [reply]
Here is my `test` blocks.
![image](https://user-images.githubusercontent.com/48143943/198287838-77bbf87c-bad7-4a1b-bc0e-9ba5d5dac619.png)
It works when i config in `package.json`: `scripts: { "test": "vitest" }` and `pnpm run test`
![image](https://user-images.githubusercontent.com/48143943/198288345-3601b125-7cbe-4ebb-843c-416e1367a93f.png)

but the same command run by global `vitest` not work.

**Comment by 1714080902120** [comment]
it worked when i used `in-source testing` but still not wrok by import....
why ?
